src/main/ipc/installer.ts

```diff
@@ -98,11 +86,47 @@ async function github(
   };
 }
 
+async function store(id: string): Promise<CheckResultSuccess | CheckResultFailure> {
+  const apiUrl =
+    ((await readTransaction("dev.replugged.Settings", (settings) => settings.get("apiUrl"))) as
+      | string
+      | undefined) ?? "https://replugged.dev";
+  const STORE_BASE_URL = `${apiUrl}/api/v1/store`;
+  const manifestUrl = `${STORE_BASE_URL}/${id}`;
+  const asarUrl = `${manifestUrl}.asar`;
+
+  const res = await fetch(`${STORE_BASE_URL}/${id}`);
```

Comment:
Unneeded code duplication.
```suggestion
  const res = await fetch(manifestUrl);
```